packages/salesforcedx-sobjects-faux-generator/src/describe/sObjectDescribe.ts

```diff
@@ -198,6 +198,25 @@ export class SObjectDescribe {
       this.accessToken = orgInfo.accessToken;
       this.instanceUrl = orgInfo.instanceUrl;
     }
+    if (!this.targetVersion) {
+      this.targetVersion = await this.getTargetApiVersion(projectPath);
+    }
+  }
+
+  private async getTargetApiVersion(projectPath: string): Promise<string> {
```

Comment:
Just wanted to let you know that this will add about 1.5-2 seconds to run time. This probably won't matter since the describe call dominates but I want you to be aware of this and if it is a worthwhile tradeoff.

Reply:
yes, it does add it once to the total.  And it will be replaced as soon as I can get the right code from the BUT-T work that Thomas, etc. are doing.   He knows about the need and is intending to do that, not there yet.
I do think it's quite important that we are consistent with the cli view of the world.